js/lex.go

```diff
@@ -24,6 +24,7 @@ const (
 	WhitespaceToken                      // space \t \v \f
 	LineTerminatorToken                  // \r \n \r\n
 	CommentToken
```

Comment:
Do we need to call this `SingleLineComment` now?

Reply:
Oh yeah good point. One thing I was afraid when writing this is that people will assume that single-line comment is always `//` and multiline is any `/*`, but this still seems like the best naming among possible, as it's literally splitting between comments that fit into one line and those that span over multiple lines.